runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -888,12 +891,12 @@ You can find help on just about any subject, by giving an argument to the
 ~~~
 # Lesson 7.2: CREATE A STARTUP SCRIPT
 
-** Enable Vim features. **
+** Enable Neovim features. **
 
-Vim has many more features than Vi, but most of them are disabled by
-default. To start using more features you have to create a "vimrc" file.
+Neovim has many more features than Vi, but most of them are disabled by default.
```

Comment:
I think we can stop referring to Vi at this point; at the same time we don't need to stress the differences to Vim here. Maybe just "Neovim is a very configurable editor. To set up your personalized configuration, you have to create..."?